### PR TITLE
different paths for MacOS v. Linux/Windows

### PR DESCRIPTION
**Pull request by mrose17**
1. use different install paths for MacOS and Linux/Windows for the `anonize2.js.mem` file

2. if `bitcoin:` URI handler not defined, then don't define `ledgerInfo.paymentURL`

auditor: @bbondy

### CODE REVIEW

**Comment by bbondy** [comment]
@diracdeltas pls review, but I'm going to merge it in anyway because packaging is broken on master.

File: app/ledger.js [hunk 1]
```diff
@@ -711,9 +711,7 @@ var updateLedgerInfo = () => {
     underscore.extend(ledgerInfo, ledgerInfo._internal.cache || {})
 
     if (typeof protocolHandler.isNavigatorProtocolHandled === 'function') {
-/* YAN: this comment is temporary until the preferences panel for payments can handle only one of these properties defined
-      delete ledgerInfo[protocolHandler.isNavigatorProtocolHandled('', 'bitcoin') ? 'buyURL' : 'paymentURL']
- */
+      if (!protocolHandler.isNavigatorProtocolHandled('', 'bitcoin')) delete ledgerInfo.paymentURL
```
Comment (diracdeltas) [inline app/ledger.js]: isn't paymentURL useful even if Brave isn't the bitcoin handler? ex: your bitcoin handler is another desktop app.

Comment (diracdeltas) [inline app/ledger.js]: if it's not useful then why show the QR code?

Comment (mrose17) [inline app/ledger.js]: some people keep the wallet on another device and use the camera on that device...

Comment (willy-b) [inline app/ledger.js]: I scan the desktop bitcoin QRs from my phone usually...

**Pull request merged**